src/utils/file_ops.cpp

```diff
@@ -28,16 +28,31 @@ bool mp::FileOps::isReadable(QDir& dir)
     return dir.isReadable();
 }
 
+bool mp::FileOps::mkpath(QDir& dir, const QString& dirName)
```

Comment:
I *think* `QDir` should be able to be `const` here.

Reply:
Indeed! Added `const`.